hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/bucket/OMBucketDeleteRequest.java

```diff
@@ -141,6 +142,16 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
             OMException.ResultCodes.BUCKET_NOT_EMPTY);
       }
 
+      // Check if bucket does not contain incomplete MPUs
+      if (omMetadataManager.containsIncompleteMPUs(volumeName, bucketName)) {
+        LOG.debug("Volume '{}', Bucket '{}' can't be deleted when it has " +
+                "incomplete multipart uploads", volumeName, bucketName);
+        throw new OMException(
```

Comment:
nit: Using String.format would be better than concat in this case. String.format would use stringbuilder internally.

Reply:
Thank you for the tip. Updated the `OMException` using `String.format`.